extensions/amp-accordion/1.0/accordion.js

```diff
@@ -25,6 +25,7 @@ import {
   useMemo,
   useState,
 } from '../../../src/preact';
+import {useStyles} from './accordion.jss';
```

Comment:
Since we do not have extensive CSS for accordion, I decided to use inline styles.  In the future if we want to use JSX, we can set it up to attach a style element to the light DOM root.